src/modules/core/components/BannedTag/BannedTag.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { HTMLAttributes } from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import { SimpleMessageValues } from '~types/index';
+import Icon from '~core/Icon';
+
+import styles from './BannedTag.css';
+
+const displayName = 'BannedTag';
+
+interface Props extends HTMLAttributes<HTMLSpanElement> {
+  text?: MessageDescriptor | string;
+  textValues?: SimpleMessageValues };
```

Comment:
There's a leftover bracket here. `{` Please remove it.

_(That's why the linter is complaining and your build is failing)_

Reply:
Fixed!